src/match/matchMakingFunction.ts

```diff
@@ -0,0 +1,83 @@
+import { MatchMakingQueueItem } from "./MatchMakingQueueItem";
+
+export const avg = (values: number[]) => {
+  return values.reduce((sum, current) => sum + current, 0) / values.length;
+};
+
+export const variance = (values: number[], waitingTime: number[]) => {
+  const waitingPeriod = 60000; // 60 seconds for now, can change latter ( wait up to 1 mins)
```

Comment:
Move these to top and all caps and underscore separate them so we know these are consts :)